root/static/scripts/common/components/ArtistCreditLink.js

```diff
@@ -70,7 +70,6 @@ const MpIcon = hydrate<MpIconProps>('span.ac-mp', (
 const ArtistCreditLink = ({
   artistCredit,
   showDeleted = true,
-  showEditsPending = true,
```

Comment:
Isn’t that supposed to be passed to `EntityLink` on [line 88](https://github.com/metabrainz/musicbrainz-server/pull/1260/commits/653075bbf021c142942fb3480376b6a799dd2da9#diff-1e1382b906771e43beecbf0a967eb48cL88) instead?

Reply:
It's already passed (as showEditsPending={!artistCredit.editsPending}) meaning this does nothing right now. If you mean we should not just do !artistCredit.editsPending there, then I don't know! :)

Reply:
This has been removed as part of GCI.